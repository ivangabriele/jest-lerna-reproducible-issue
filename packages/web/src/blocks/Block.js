import styled from "@emotion/styled";
import React from "react";

const Container = styled.div`
  color: blue;
`;

const Block = ({ text }) => <Container>{text}</Container>;

export default Block;
