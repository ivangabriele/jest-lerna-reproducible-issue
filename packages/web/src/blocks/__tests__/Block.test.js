import React from "react";
import TestRenderer from "react-test-renderer";

import Block from "../Block";

describe("blocks/Block", () => {
  it("should render", () => {
    const $block = TestRenderer.create(<Block />);

    expect($block).toMatchSnapshot();
  });
});
