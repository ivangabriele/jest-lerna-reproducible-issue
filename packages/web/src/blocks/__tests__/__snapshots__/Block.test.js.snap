// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blocks/Block should render 1`] = `
.emotion-0 {
  color: blue;
}

<div
  className="emotion-0 emotion-1"
/>
`;
